app.lua

```diff
@@ -126,8 +134,18 @@ function app:handle_404()
 end
 
 function app:handle_error(err, trace)
+    -- self.current_user is not available here.
+    local current_user = package.loaded.Users:find(self.session.username)
+    if current_user then
+        user_params = current_user:rollbar_params()
+    else
+        user_params = {}
+    end
     print(err)
     print(trace)
+    rollbar.set_person(user_params)
+    rollbar.set_custom_trace(err .. "\n\n" .. trace)
+    rollbar.report(rollbar.ERR, helpers.normalize_error(err))
```

Comment:
There are a few supported levels. We are free to call this wherever, which can be useful if we need to check on some other behavior.

Levels:
```
  CRIT  = 'critical',
	  ERR   = 'error',
	  WARN  = 'warning',
	  INFO  = 'info',
	  DEBUG = 'debug',
```